test/LeanCode.IntegrationTests/TimestampTzTests.cs

```diff
@@ -0,0 +1,67 @@
+using LeanCode.DomainModels.Model;
+using LeanCode.IntegrationTests.App;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Xunit;
+
+namespace LeanCode.IntegrationTests;
+
+[System.Diagnostics.CodeAnalysis.SuppressMessage("?", "CA1001", Justification = "Disposed with `IAsyncLifetime`.")]
+public class TimestampTzTests : IAsyncLifetime
+{
+    private readonly TestApp app;
+
+    public TimestampTzTests()
+    {
+        app = new TestApp();
+    }
+
+    [PostgresFact]
+    public async Task TimestampTz_local_timestamp_access_is_correctly_translated()
+    {
+        using var scope = app.Services.CreateScope();
+        var dbContext = scope.ServiceProvider.GetRequiredService<TestDbContext>();
+        var today = DateOnly.FromDateTime(DateTime.UtcNow);
```

Comment:
I'm not sure we should rely on "current date" in tests - although we don't really depend on the exact date (hopefully), making it dynamic might make changes more tricky.

Reply:
It indeed shouldn't matter (because on any given day the difference between Los Angeles and Tokyo local times should be greater than a few hours). But I can hardcode any arbitrary date here if you believe it will be an improvement.

Reply:
Most of the people are used to hardcoded data in tests (for the "normal" unit test reasons) and I think it would make understanding what is going on easier in that case - that's the only benefit I see. So basically your call which way we go.

Reply:
I'll just hardcode it.